### PR TITLE
[CIAM-2134] Documentation for session-limits-feature

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

**Review by mposolda — CHANGES_REQUESTED** [review]
@douglaspalmer Thanks for the PR!

Is it possible to include also small note to the release notes - it is in the file 18_0_0.adoc. Please include "Thanks to" with the contributor, who sent original PR :-)

**Review by mposolda — DISMISSED** [review]
@douglaspalmer Thanks!

**Review by mposolda — DISMISSED** [review]

**Comment by mposolda** [comment]
@andymunro Can you please review this?

File: server_admin/topics/authentication/flows.adoc [hunk 1]
```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
 one of the specified levels. If it is not able to return one of the specified levels (For example if the requested level is unknown or bigger than configured conditions
 in the authentication flow), then {project_name} will throw an error.
 
+[[_user_session_limits]]
+*User session limits*
+
+Limits on the number of session that a user can have can be configured. Sessions can be limited per realm or per client.
+
+To add session limits to a flow perform the following steps.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: Add a comma after flow (this in an introductory phrase)

File: server_admin/topics/authentication/flows.adoc [hunk 2]
```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
...
+
+. Click *Add execution* for the flow.
+. Select *User Session Count Limiter* from the item list.
+. Click *Save*.
+. Click *Required* for the *User Session Count Limiter* authentication type to set its requirement to required.
+. Click *Actions* for the *User Session Count Limiter*.
+. Click *Config*.
+. Enter an alias for this config.
+. Enter the required maximum number of sessions a user can have in this realm. If a value of 0 is used then this check is disabled.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: should be 
If a value of 0 is used, this check is disabled.

" then" is not needed. Also, a comma should conclude an introductory phrase.

File: server_admin/topics/authentication/flows.adoc [hunk 3]
```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
...
+. Enter the required maximum number of sessions a user can have for the client. If a value of 0 is used then this check is disabled.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: If a value of 0 is used, this check is disabled.

File: server_admin/topics/authentication/flows.adoc [hunk 4]
```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
...
+. Select the behavior that is required when the user tries to create a session after the limit is reached. Available bahaviors are:
+
+  Deny new session - when a new session is requested and the session limit is reached no new sessions can be created.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: add a comma after reached
(introductory phrase)
remove "new" (unnecessary word)

File: server_admin/topics/authentication/flows.adoc [hunk 5]
```diff
@@ -406,6 +406,34 @@ Note when the login request initiates a request with the `claims` parameter requ
...
+  Terminate oldest session - when a new session is requested and the session limit has been reached the oldest session will be removed and the new session created.
```
Comment (andymunro) [inline server_admin/topics/authentication/flows.adoc]: add a comma after reached

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good @mposolda. Just a few suggestions for changes.

**Review by mposolda — APPROVED** [review]
@douglaspalmer @andymunro Thanks for the review and fixes!

**Pull request merged**